main.go

```diff
@@ -183,8 +172,17 @@ func main() {
 
 	keycloakProfileService := login.NewKeycloakUserProfileClient()
 
+	// Try to fetch the initial list of clusters and start Cluster Service cache refresher
+	err = clusterservice.Start(context.Background(), config)
+	if err != nil {
+		// It's not a critical error. Cluster management service can be offline. We will try to fetch clusters later when we need them.
+		log.Warn(nil, map[string]interface{}{
+			"err": err,
+		}, "failed to fetch clusters")
```

Comment:
just want to know, why it's not critical?

Reply:
is there any risk of error if we load a user info but can't associate the user's account with a cluster in the cache?

Reply:
There is one usecase when it's ok when we fail here: Cluster management service is not started yet because it's wait for Auth service to start to fetch public keys.
If we required Cluster service to be ready to start Auth then we would create cycle dependency when both Auth and Cluster wait for each other to start up.

If cluster loading fails in main then we will try to load it again later when we actually need it:
- when user tries to link OSO-OSIO accounts
- when user tries to sign-up

If Cluster management still down then users will see an error message.

Reply:
thanks for clarification and this should be captured somewhere else if possible.  may be in code itself ??

Reply:
Add more detailed comment in https://github.com/fabric8-services/fabric8-auth/pull/659/commits/ff60248226b5ca67e651abf7ce65d43a3994efc3